docs/migrations/index.md

```diff
@@ -0,0 +1,15 @@
+(migrations-page)=
+
+# Migration notes
+
+The following pages contain migration notes for each stable release of Nextflow:
+
+```{toctree}
+:maxdepth: 1
+
+25-04
+strict-syntax
+24-10
+24-04
+dsl1
```

Comment:
Could drop this out from this list and have another introduction explaining DSL1 is out of date.

Reply:
Okay. I think I'll just leave it as is